lib/common/formats.js

```diff
@@ -239,6 +240,21 @@ module.exports = {
     return fileHeader(this.options, 'short') + iconsWithPrefix('@', dictionary.allProperties, config, 'short');
   },
 
+  /**
+   * Creates a Stylus file with variable definitions based on the style dictionary
+   *
+   * @memberof Formats
+   * @kind member
+   * @example
+   * ```stylus
+   * $color-background-base= #f0f0f0;
+   * $color-background-alt= #eeeeee;
+   * ```
```

Comment:
Although stylus does not require trailing semicolons, it does no harm. If we want to be more concise we could also remove them, but for the cost of adding another parameter to the `variablesWithPrefix` function.